src/components/react-native-image-viewing/dist/ImageViewing.js

```diff
@@ -159,24 +156,24 @@ function ImageViewing({
             <View>
               {/* Your overlay content */}
 
-              <ImageItem
+              {!playingUri && <ImageItem
                 onZoom={onZoom}
                 imageSrc={imageSrc}
                 onRequestClose={onRequestCloseEnhanced}
                 onLongPress={onLongPress}
                 delayLongPress={delayLongPress}
                 swipeToCloseEnabled={swipeToCloseEnabled}
                 doubleTapToZoomEnabled={doubleTapToZoomEnabled}
-              />
-              {isVideoButton && (
-                <TouchableOpacity
-                  style={styles.playButton}
-                  onPress={playVideoFullScreen}
-                >
+              />}
+              {isVideoButton && !playingUri &&(
+              <TouchableOpacity
+              style={styles.playButton}
+              onPress={playVideoFullScreen}
+              >
                   <SvgXml xml={playBtn} width="50" height="50" />
-                </TouchableOpacity>
+              </TouchableOpacity>
               )}
-            </View>
+                                     </View>
           )}
           onMomentumScrollEnd={onScroll}
           //@ts-ignore
```

Comment:
If it not hard, please try to remove this `ts-ignore`

Reply:
sure sure